### ISSUE TITLE
Edit "Pickup Time" Title and Label

### ISSUE THREAD

**Issue by Stix7** [mattbanks/woocommerce-local-pickup-time#59]
Hi,

How do I change the "**Pickup Time**" text to "**Pickup/Delivery Time**"?

![change-label](https://user-images.githubusercontent.com/50157792/60989175-120ee300-a2fa-11e9-813c-a4ffdcc34a44.png)

All customers can order pickup, but only a select few business customers can order pickup or delivery. We have disabled the global shipping feature. and added a dropdown option with pre-qualified business addresses. So, only need to change the title and label text to "Pickup/ Delivery Time".

**Comment by timnolte** [comment]
@stix7 sorry, I thought that I responded on this. Perhaps you found a workaround to setup your own translation file for the plugin which would allow you to change this text. I will say that the plugin target was originally for pickup as there are other plugins that do both. Well actually all other plugins I looked at forced you to have both and those that need pickup strictly didn't have many options.
